src/lwc/geFormField/geFormField.js

```diff
@@ -340,6 +356,30 @@ export default class GeFormField extends LightningElement {
         }
     }
 
+    // when using lightning-lookup-field, instead of binding to the Data Import fields or donor information
+    // we bind to Opportunity.AccountId / Opportunity.Primary_Contact__c
+    get lookupFieldApiName() {
+        if (this.objectApiName === DATA_IMPORT.objectApiName) {
+            if (this.fieldApiName === ACCOUNT1_IMPORTED.fieldApiName) {
```

Comment:
These fields aren't really mapped, they're hardcoded as part of the BDI process. See [BDI_Donations.cls lines 390-404](https://github.com/SalesforceFoundation/NPSP/blob/9ef61dced9b667542b260bd48ae4725c5d1d06f0/src/classes/BDI_Donations.cls#L390-L404).

Remapping should not affect this either, when the form asks for the value of this field it still gets the Data Import (source) field and value back in response instead of these remapped AccountId/Primary_Contact__c (target) fields back.

Reply:
Ah - I read you! This tracks, thanks for explaining!